### ISSUE TITLE
bug: double space

### ISSUE THREAD

**Issue by oprogramador** [adrianheine/node-in-words#6]
It is:
```
> require('in-words').en(1010)
'one thousand  ten'
```

It should be: `one thousand ten` (with a single space between `thousand` and `ten`)

`in-words` version: 0.2.2

It works correctly for Polish (a single space) and German (no space).

**Comment by adrianheine** [comment]
Thanks for the report!

**Comment by oprogramador** [comment]
I see it's fixed now (`in-words@0.2.3`). Thanks

```
> require('in-words').en(1010)
'one thousand ten'
```
